agent/unit-test/containeroptionparser_unittest.cpp

```diff
@@ -58,11 +58,45 @@ TEST_F(ContainerOptionParserTest, parseManyTimes) {
     ASSERT_TRUE(m_options->enableWriteBuffer());
 }
 
-TEST_F(ContainerOptionParserTest, parseConfigNice2) {
+TEST_F(ContainerOptionParserTest, parseConfigNiceDisabled) {
     ASSERT_NO_THROW(parse("[{\"enableWriteBuffer\": false}]"));
     ASSERT_FALSE(m_options->enableWriteBuffer());
 }
 
+TEST_F(ContainerOptionParserTest, parseConfigNiceWithTmpfsAndSize) {
```

Comment:
I think all tests should have a comment what they test, what the values mean, what the purpose is etc. Please comment the tests. If it's deemed overkill to have a comment on each, unlikely IMO, then at least a comment before the batch of functions that explains the overall strategy.